rokka/organizations.go

```diff
@@ -0,0 +1,24 @@
+package rokka
+
+type OrganizationResponse struct {
+	ID           string `json:"id"`
+	DisplayName  string `json:"display_name"`
+	Name         string `json:"name"`
+	BillingEmail string `json:"billing_email"`
+	Limit        struct {
+		SpaceInBytes   *int `json:"space_in_bytes"`
+		TrafficInBytes *int `json:"traffic_in_bytes"`
+	} `json:"limit"`
```

Comment:
is that how `go fmt` formats it? looks weird 😮

Reply:
go fmt zäme